app/html/shared/sidebar/home.html

```diff
@@ -10,6 +10,8 @@
 <hr>
 <a href="@{url_for('home.view_subs')}" class="sbm-post pure-button">@{_('View all subs')}</a>
 <a href="@{url_for('subs.random_sub')}" class="sbm-post pure-button">@{_('Go to random sub')}</a>
+@end
+@if config.site.sub_creation_admin_only == True and current_user.is_admin():
```

Comment:
This condition would always hide the button if `site.sub_creation_admin_only` is False

Reply:
oops good point - will fix!